{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "16496100_f8428f72",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 4
      },
      "lineNbr": 0,
      "author": {
        "id": 1002532
      },
      "writtenOn": "2024-02-05T09:36:17Z",
      "side": 1,
      "message": "Python also doesn\u0027t consider relative span across different drives valid.\n\n\u003e\u003e\u003e import os\n\u003e\u003e\u003e os.path.relpath(\u0027C:/a/b/c\u0027, \u0027C:/a/b/c/d/e\u0027)\n\u0027..\\\\..\u0027\n\u003e\u003e\u003e os.path.relpath(\u0027C:/a/b/c\u0027, \u0027C:/a/b/c/v/e\u0027) \n\u0027..\\\\..\u0027\n\u003e\u003e\u003e os.path.relpath(\u0027C:/a/b/c\u0027, \u0027C:/a/e/c/v/e\u0027) \n\u0027..\\\\..\\\\..\\\\..\\\\b\\\\c\u0027\n\u003e\u003e\u003e os.path.relpath(\u0027C:/a/b/c\u0027, \u0027D:/a/e/c/v/e\u0027) \nTraceback (most recent call last):\n  File \"\u003cstdin\u003e\", line 1, in \u003cmodule\u003e\n  File \"\u003cfrozen ntpath\u003e\", line 754, in relpath\nValueError: path is on mount \u0027C:\u0027, start on mount \u0027D:\u0027",
      "revId": "c1e835b83b1d9336ac4c7486a19f3efa7d569117",
      "serverId": "c360c8e5-7aa8-3905-a7a0-810934766dac"
    }
  ]
}